vnc.html

```diff
@@ -91,6 +91,8 @@
             <div>noVNC encountered an error:</div>
             <br>
             <div id="noVNC_fallback_errormsg"></div>
+            <div>A common cause of errors is browser cache.</div>
```

Comment:
I'm sceptical about this. We've seen this for various other things. And if we start recommending deployments with stricter cache control then it should cease to be an issue.